schemas/string_interpolation/definitions.json

```diff
@@ -173,6 +173,11 @@
         "type": "string",
         "enum": ["same_name_items"]
       },
+      "selector": {
+        "type": "string",
+        "description": "Address answer type field selector",
+        "enum": ["line1", "line2", "town", "postcode"]
+      },
```

Comment:
I am a bit torn on this as the idea was to add support for composite answer values not just addresses. Given you already check specifically the address answer type in code, the "selectors" that can be used could live in code which will make this more flexible.